src/scripts/run_backup.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
I don't really understand why a bash script is needed here, we can run 2 commands directly from python.
in any case, can be handled as a separate ticket/PR.

Reply:
addressed in [2f28871](https://github.com/canonical/mysql-k8s-operator/pull/136/commits/2f288712cb675a55668e3c75896aaba6feebe24d)